db/schema.rb

```diff
@@ -67,4 +147,5 @@
   add_foreign_key "deals", "users", column: "borrower_id"
```

Comment:
we dont use activerecord schema anymore.  These lines below are they generated?

Reply:
these are auto-generated